.changelog/3677.changed.txt

```diff
@@ -0,0 +1 @@
+feat: use Sumo Logic repository for all images
```

Comment:
We should probably have a call out of this change in the release notes. It's not breaking, but it might be surprising for users who rehost the images, have security scanner exceptions set for specific repositories, and so on.

Reply:
I will add it to changelog, as I'm going to relase it 😄